resources/web/panel/js/pages/giveaways/auction.js

```diff
@@ -34,6 +34,10 @@ $(run = function () {
             })).append('&nbsp; Close').removeClass('btn-success').addClass('btn-warning');
         }
 
+        if (e['extTime'] !== undefined) {
+            $('#auction-extTime').val(e['extTime']);
```

Comment:
Codacy has a fix for the issue: [Replace `··$('#auction-extTime').val(e['extTime'` with `$("#auction-extTime").val(e["extTime"`](https://app.codacy.com/gh/Psychoboy/PhantomBot/pullRequest?prid=9180768)
```suggestion
          $("#auction-extTime").val(e["extTime"]);
```